config/watcher_local_test/kustomization.yaml

```diff
@@ -44,48 +44,53 @@ patches:
     target:
       kind: Deployment
 
+# Non-Native k8s resources
 patchesJson6902:
```

Comment:
can you run kustomize edit fix on all this?

Reply:
I did it, but this actuelly destroyed the kustomize and made it not working anymore